src/openrct2/world/Sprite.h

```diff
@@ -23,6 +23,10 @@ union rct_sprite
 {
     uint8_t pad_00[0x200];
     EntityBase base;
+    rct_sprite()
```

Comment:
Maybe add a comment as to why this is needed and what is required for it to be removed?